tsconfig.json

```diff
@@ -31,6 +31,7 @@
     "packages/blockchain-info-components/src/",
     "packages/blockchain-wallet-v4-frontend/src/",
     "packages/blockchain-wallet-v4/src/",
+    "*/types.ts",
```

Comment:
whats this do?

Reply:
I was hoping it would fix the webpack watch issues, but it doesnt. Ill remove